executor/window.go

```diff
@@ -202,7 +202,7 @@ type aggWindowProcessor struct {
 
 func (p *aggWindowProcessor) consumeGroupRows(ctx sessionctx.Context, rows []chunk.Row) ([]chunk.Row, error) {
 	for i, windowFunc := range p.windowFuncs {
-		err := windowFunc.UpdatePartialResult(ctx, rows, p.partialResults[i])
+		_, err := windowFunc.UpdatePartialResult(ctx, rows, p.partialResults[i])
```

Comment:
How about leaving a TODO comment here since memory usage here should also be traced?

Reply:
done